rs/types/types/src/messages/ingress_messages.rs

```diff
@@ -282,6 +282,10 @@ impl SignedIngress {
         self.signed.content()
     }
 
+    pub fn signed(&self) -> &HttpRequest<SignedIngressContent> {
+        &self.signed
+    }
+
```

Comment:
Just stumbled upon
```
impl AsRef<HttpRequest<SignedIngressContent>> for SignedIngress {
    fn as_ref(&self) -> &HttpRequest<SignedIngressContent> {
        &self.signed
    }
}
```
so theoretically, the new `signed(&self)` is not strictly necessary as the same can be achieved with the existing `as_ref()`. For people who don't know this code well (like me), this is not obvious, however.